apps/daimo-mobile/.gitignore

```diff
@@ -10,6 +10,8 @@ npm-debug.*
 *.orig.*
 web-build/
 
+.env*
```

Comment:
nit: can delete this file from diff

renamed `.sample-env` to `.env.example` to match other apps (web, clippy, ...)